yt/frontends/eagle/fields.py

```diff
@@ -26,10 +26,9 @@ class EagleNetworkFieldInfo(OWLSFieldInfo):
          'Fe17', 'Fe18', 'Fe19', 'Fe20', 'Fe21', 'Fe22', 'Fe23',\
          'Fe24', 'Fe25', 'Fe25', 'Fe27',)
 
-    def __init__(self, *args, **kwargs):
-        
-        super(EagleNetworkFieldInfo,self).__init__( *args, **kwargs )
-        
+    def __init__(self, ds, field_list, slice_info=None):
```

Comment:
We probably don't need this if it's empty, I think.  But not necessary to remove it.